addons/event/static/src/client_action/event_barcode.js

```diff
@@ -76,25 +76,21 @@ export class EventScanView extends Component {
         }
     }
 
+    getEventAvailableViews() {
```

Comment:
seems you forgot to remove that part :slightly_smiling_face:

Reply:
yeah hihi